src/components/TileCategorySelection/index.jsx

```
@@ -0,0 +1,143 @@
+              <Trans i18nKey="all">All</Trans>
+            </MenuItem>
+          )}
+          {validGroupFilter &&
+            groups &&
+            groups[validGroupFilter] &&
+            Object.entries(groups[validGroupFilter].intensities || {})
+              .sort(([a], [b]) => Number(a) - Number(b))
+              .map(([intensity, count]) => (
+                <MenuItem key={intensity} value={Number(intensity)}>
+                  {mappedGroups &&
+                  mappedGroups[gameMode] &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)